src/modules/settings/BlockedSkills.svelte

```diff
@@ -0,0 +1,226 @@
+<script>
+import getValueJSON from '../system/getValueJSON';
+import setValue from '../system/setValue';
+import {
+  checkSkill,
+  clearCheckedSkills,
+  getCheckedSkills,
+  submitSkillChanges,
+} from './blockedSkills';
+
+let blockedSkillLists = getValueJSON('blockedSkillLists');
+let newListName = '';
+let error = false;
+let listName = '';
+
+function checkForDuplicates(list) {
```

Comment:
this function can be made pure and moved out as an import by having blockedSkillLists as a parameter.